packages/story-editor/src/components/canvas/singleSelectionMoveable/useResize.js

```diff
@@ -146,6 +150,9 @@ function useSingleSelectionResize({
     if (handleElementOutOfCanvas(target)) {
       return;
     }
+    if (handleFullbleedMediaAsBackground(target)) {
```

Comment:
We might need to add this after updating the selected element: otherwise, if the user clicks Undo, the pre-resizing state will be restored, however, only setting as background should be reverted.

Reply:
I was wondering to what state it should undo because when you drop media as BG it will undo to a different state (post-move/pre-bg, and not to the initial state which was pre-move [state the video started with]):

https://user-images.githubusercontent.com/21036224/152511777-fdbc618d-b6f4-4343-af8e-153d93045147.mov

Is that drop as BG undo state correct or not?

Reply:
> Is that drop as BG undo state correct or not?

Yep, that should be correct (if I understand correctly) -- the move / resizing stays but just setting the background is undone -- that's the information I have from UX at least.

The only oddity with this will be that every time the user moves it again, it will be set as background again as long as it covers the whole page. 🤔

Reply:
Will this be ok? 86407ba5ab8b34b04e53f628068868e3c3e72a47

https://user-images.githubusercontent.com/21036224/152532863-0f4c3703-ede1-4817-b08b-b60d0505e537.mov

Reply:
Can't see how many times you click on the Undo button but if I understand correctly then yes -- this should be correct -- when clicking "Undo" the first time, the media will stay larger than the Page but not be background anymore.

Reply:
> when clicking "Undo" the first time, the media will stay larger than the Page but not be background anymore.

✅